streaming/java/streaming-api/src/main/java/org/ray/streaming/plan/PlanBuilder.java

```diff
@@ -44,7 +45,7 @@ private void processStream(Stream stream) {
       processStream(parentStream);
     } else if (stream instanceof StreamSource) {
       planVertex = new PlanVertex(vertexId, parallelism, VertexType.SOURCE, streamOperator);
-    } else if (stream instanceof DataStream) {
+    } else if (stream instanceof DataStream || stream instanceof PythonDataStream) {
```

Comment:
Should they share the same base interface?

Reply:
The class hierarchy is:
```
Stream
-- DataStream
    -- DataStreamSource
    -- DataStreamSink
-- PythonDataStream
    -- PythonDataSource
    -- PythonDataSink
```
They share base interface `Stream`, but `Stream` is base interface of all streams.